pkg/addonmanager/controllers/agentdeploy/hosted_hook_sync_test.go

```diff
@@ -214,7 +214,7 @@ func TestHostingHookReconcile(t *testing.T) {
 			},
 		},
 		{
-			name: "deploy hook manifest for a deleting addon with 2 finalizer, completed",
+			name: "deploy hook manifest for a deleting addon with 2 finalizer, completed,without completed condition",
```

Comment:
the name is confusing, I do not know if it is complete or not.

Reply:
done